pkg/controller/worker/genericactuator/actuator_reconcile.go

```diff
@@ -142,7 +142,7 @@ func (a *genericActuator) Reconcile(ctx context.Context, worker *extensionsv1alp
 	}
 
 	// Wait until all generated machine deployments are healthy/available.
-	timeoutCtx, cancel := context.WithTimeout(ctx, 30*time.Minute)
+	timeoutCtx, cancel := context.WithTimeout(ctx, 15*time.Minute)
```

Comment:
With gardener/gardener#1364 we now have:
```golang
const WorkerDefaultTimeout = 5 * time.Minute
```

We wait 5min for worker to be `Ready`, and the worker waits 15 min for its machdeploys to be `Available`?
The same question applies also for the deletion.

Reply:
Hm, yeah, good point, thank you. Set it to `5m` now, if it doesn't complete within the `5m` then it will be retried by the controller anyway and another `5m` will be waited. However, if an error occurred, it will now be propagated to the user very fast.